contracts/ColonyStorage.sol

```diff
@@ -29,6 +29,10 @@ contract ColonyStorage is DSAuth {
   // this one will have the getters. Make custom getters in the contract that seems most appropriate,
   // and add it to IColony.sol
 
+  uint8 constant MANAGER = 0;
+  uint8 constant EVALUATOR = 1;
+  uint8 constant WORKER = 2;
+
```

Comment:
Because of the `EtherRouter` design the ordering of storage variables here shouldn't be interrupted and the `resolver` address property has to always be the first one. Please append these new storage constants to the end of variable declarations at line https://github.com/JoinColony/colonyNetwork/pull/171/files#diff-f4e9c4137ebcc35fead90e9b07c10f4dR73

Reply:
Maybe it's not even needed we move these to Storage contract

Reply:
Right now they are used in ColonyFunding and ColonyTask, but since functions in ColonyFunding that are using these variables are probably going to be removed, i can revert this part if you want

Reply:
No, it's fine. Let's leave them there